File: c/006/main.c
/*
 * ProjectEuler 006
 * C / main.c
 * copyright (c) 2015 Susisu
 *
 * GCC / Apple LLVM version 6.0 (clang-600.0.56) (based on LLVM 3.5svn)
 * `gcc main.c -o bin/main`
 */

#include <stdio.h>

int main() {
    int s = 0;
    int t = 0;
    for (int n = 1; n <= 100; n++) {
        s += n;
        t += n * n;
    }
    printf("%d\n", s * s - t);
    return 0;
}

File: scala/006/Main.scala
/*
 * ProjectEuler 006
 * Scala / Main.scala
 * copyright (c) 2015 Susisu
 *
 * Scala 2.11.4
 * `scalac Main.scala -d bin`
 */

object Main {
    def main(args: Array[String]) {
        var s = 0;
        var t = 0;
        for (n <- 1 to 100) {
            s += n;
            t += n * n;
        }
        println(s * s - t);
    }
}
